src/highlevel.jl

```diff
@@ -57,7 +57,12 @@ end
 SoapySDRDevice_unmake(d::Device) = SoapySDRDevice_unmake(d.ptr)
 Base.cconvert(::Type{<:Ptr{SoapySDRDevice}}, d::Device) = d
 Base.unsafe_convert(::Type{<:Ptr{SoapySDRDevice}}, d::Device) = d.ptr
-Base.close(d::Device) = SoapySDRDevice_unmake(d)
+function Base.close(d::Device)
+    SoapySDRDevice_unmake(d)
```

Comment:
I wonder if these shouldn't be `unsafe_close!` calls that only get called by finalizers, because otherwise it's possible to make the library segfault unless all functions that operate on a `Device` (the same applies to `Stream`) checks `isopen` first.

Reply:
I've noticed this as well some times in the REPL. Opened #44 to track.